justfile

```diff
@@ -79,12 +79,24 @@ win-arm64-ring16 := if target == "aarch64-pc-windows-msvc" { " --config='patch.c
 # MIR optimisation level (defaults to 2, bring it up to 4 for release builds)
 rustc-miropt := if for-release != "" { " -Z mir-opt-level=4" } else { "" }
 
+# Use rust-lld that is bundled with rustup to speedup linking
+# and support for icf=safe.
+#
+# -Zgcc-ld=lld uses the rust-lld that is bundled with rustup.
+#
+# TODO: There is ongoing effort to stablise this and we will need to update
+# this once it is merged.
+# https://github.com/rust-lang/compiler-team/issues/510
+rust-lld := if target-os != "windows" { " -Zgcc-ld=lld" } else { "" }
```

Comment:
*stabilise

also let's use `-Z gcc...` (mind the space) in keeping with the style of the others

Reply:
Thanks!

I've fixed the typo and the style.